routers/api/scouting.ts

```diff
@@ -33,7 +33,9 @@ import {
     getTeam
 } from "../../helpers/tba";
 import scoutingConfig from "../../config/scouting";
-import { processAdmin } from "../../helpers/adminHelpers";
+import ScoutingEntry from "../../models/scoutingEntry";
+import Team from "../../models/team";
```

Comment:
should use helpers rather than importing models directly (replicate functionality in the helpers/scouting.ts and import those functions here)